src/main/java/org/codeforamerica/messaging/models/TemplateSet.java

```diff
@@ -0,0 +1,32 @@
+package org.codeforamerica.messaging.models;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.*;
+import org.codeforamerica.messaging.utils.RegexPatternStrings;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.OffsetDateTime;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString(onlyExplicitlyIncluded = true)
+public class TemplateSet {
```

Comment:
IMO, the Set suffix does not add value. When we have endpoints for these, I imagine them being "/templates" and not "/templateSets". I think we are better off calling them just templates, which have language, experiment and version variants.